services/graylog/scripts/alerts.template.yaml

```diff
@@ -271,12 +271,12 @@
     grace_period_ms: 0
     backlog_size:
   alert: true
-- title: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected. Please investigate."
-  description: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected. Please investigate."
+- title: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected."
+  description: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected. Human intervention required. Please investigate."
   priority: 3
   config:
     query: >
-      "unexpectedly failed [OEC" AND container_name:/.*director-v2.*/
```

Comment:
Q: why not to keep the OEC-like errors?

Reply:
Because there were >150 (for all 6 deployments summed) in 24h, so this is too spammy for an "escalation to human" at this point :(

Once we have few enough OECs that one popping up can be categorized as "unexpected", we should indeed have the OEC alert again ;)